### PR TITLE
QBT remove false link to torrent folder.

### PR DESCRIPTION
**Pull request by BKSteve**
Fixes a QBT folder error where BlackHole path passed to QBT rather than using QBT defined folders.
https://discord.com/channels/502612977271439372/1173806065427697775

## Summary by CodeRabbit

- **New Features**
  - Enhanced qBittorrent integration with new options for torrent path and incomplete path settings.
  - Improved file browser visibility for torrent path selection.

- **Refactor**
  - Updated logging practices for better clarity and debugging, including a shift from `info` to `debug` level and adoption of f-string formatting.

- **Bug Fixes**
  - Fixed issues with torrent path settings to align with user-configured directories.

### CODE REVIEW

**Comment by miigotu** [comment]
I don't agree with this. I think that the folder needs to be shown in the js, where we hide and show different settings depending on what client it is. If they don't want it in a folder either they should empty that field, or we need to add an option to set whether they want to use labeler or our path setting . This was a new feature, we weren't able to send a path in the past. I'm curious if there is an incomplete_path option for qBitTorrent API also, it would be nice to be able to pass that.

**Comment by miigotu** [comment]
Need to change this line: https://github.com/SickChill/sickchill/blob/fa39f385b7edf8247b4d24f0b54eb1f114249435/sickchill/gui/slick/js/core.js#L1931

```diff
--- $('#torrent_path_option').hide();
+++ $('#torrent_path_option').show();
+++ $('#torrent_path_option').find('.fileBrowser').show();
```

What I'm also curious about is if qBitTorrent xmlrpc can accept a `$('#torrent_path_incomplete_option').show()`

**Comment by miigotu** [comment]
torrent_path_incomplete_option can be coded in for qBitTorrent:
Screenshot_20231120-000310

**Comment by BKSteve** [comment]
Yes. That's a more complete solution giving the options. The new other torrent items can probably do this too. 
Will do a more detailed push after review

**Comment by BKSteve** [comment]
`save_path` can be set, 
setting `temp_path` no success, ~~if setting enable in qbt or not, still unable to get qbt to accept temp location. Looks to be a read only parameter or I'm doing something wrong.~~

Add: https://qbittorrent-api.readthedocs.io/en/latest/apidoc/torrents.html
`download_path` test soon.

**Comment by BKSteve** [comment]
clean up too many info logs for DownloadStation, converted to debug.

File: sickchill/oldbeard/clients/qbittorrent.py
Comment (BKSteve) [inline sickchill/oldbeard/clients/qbittorrent.py:1]: add this debug logging to others?

**Comment by BKSteve** [comment]
@coderabbitai review

**Comment by BKSteve** [comment]
@coderabbitai pause

File: sickchill/oldbeard/clients/qbittorrent.py [hunk 1]
```diff
@@ -41,20 +41,38 @@ def test_client_connection(self):
...
+        logger.debug(f"Posting as file with {self.__torrent_args(result)}")
+        action = self.api.torrents_add(torrent_files=[result.content], **self.__torrent_args(result))
+        self.__add_trackers(result)
+        return action
 
     def _set_torrent_priority(self, result: "TorrentSearchResult"):
         if not self.api.app_preferences().get("queueing_enabled"):
             return True
-        return (self.api.torrents_decrease_priority, self.api.torrents_increase_priority)[result.priority == 1](result.hash.lower)
+        return (self.api.torrents_decrease_priority, self.api.torrents_increase_priority)[result.priority == 1](result.hash.lower())
```
Comment (miigotu) [inline sickchill/oldbeard/clients/qbittorrent.py:77]: These missing parenthesis are a really old bug o can't believe I missed this all those years.

File: sickchill/oldbeard/clients/qbittorrent.py
Comment (miigotu) [reply sickchill/oldbeard/clients/qbittorrent.py:1]: Sure, this was added though when their API changes between dsm5 and 6 I think though. Up to you.

Comment (BKSteve) [reply sickchill/oldbeard/clients/qbittorrent.py:1]: I meant add `logging.debug` for uT and rT etc. so there is some data if any of them aren't working.

Comment (miigotu) [reply sickchill/oldbeard/clients/qbittorrent.py:1]: By all means, if you think it might be helpful at another time for debugging go for it.

**Pull request merged**